configurator/src/config.template

```diff
@@ -4,6 +4,12 @@ bitcoin-rpcpassword={bitcoin_rpc_pass}
 bitcoin-rpcconnect={bitcoin_rpc_host}
 bitcoin-rpcport={bitcoin_rpc_port}
 plugin=/usr/local/libexec/c-lightning/plugins/c-lightning-http-plugin
+plugin=/usr/local/libexec/c-lightning/plugins/rebalance/rebalance.py
+plugin=/usr/local/libexec/c-lightning/plugins/summary/summary.py
+plugin=/usr/local/libexec/c-lightning/plugins/c-lightning-REST/plugin.js
+rest-port=3001
+rest-docport=4001
+rest-protocol=https
```

Comment:
this is listening presumably on a different interface than the original rest plugin on line 6, should we deprecate the one on line 6, would that destroy spark's ability to work?

Reply:
Leaving both plugins in for now.